TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -199,9 +273,12 @@ private static Parameters parseArgs(String[] args) {
     private static String getTransformerConfig(Parameters params) {
         if (params.transformerConfigFile != null
             && !params.transformerConfigFile.isBlank()
+            && params.transformerConfigEncoded != null
+            && !params.transformerConfigEncoded.isBlank()
             && params.transformerConfig != null
             && !params.transformerConfig.isBlank()) {
-            System.err.println("Specify either --transformer-config or --transformer-config-file, not both.");
+            System.err.println("Specify only one of --transformer-config-base64, --transformer-config or " +
```

Comment:
This is currently only occurring if all 3 are specified, let's make sure this occurs if we have 2/3 specified